common/reviews/api/load-themed-styles.api.ts

```diff
@@ -0,0 +1,60 @@
+// @public
+enum ClearStyleOptions {
+  // (undocumented)
+  all = 3,
+  // (undocumented)
+  onlyNonThemable = 2,
+  // (undocumented)
+  onlyThemable = 1
+}
+
+// @public
+export function clearStyles(option: ClearStyleOptions = ClearStyleOptions.all): void;
+
+// @public
+export function configureLoadStyles(loadStylesFn: ((processedStyles: string, rawStyles?: string | ThemableArray) => void) | undefined): void;
+
+// @public
+export function configureRunMode(mode: Mode): void;
+
+// @public
+export function detokenize(styles: string | undefined): string | undefined;
+
+// @public
+export function flush(): void;
+
+// @public (undocumented)
+interface ITheme {
+  // (undocumented)
+  [ key: string ]: string;
+}
+
+// @public (undocumented)
+interface IThemingInstruction {
+  // (undocumented)
+  defaultValue?: string;
+  // (undocumented)
+  rawString?: string;
+  // (undocumented)
+  theme?: string;
+}
```

Comment:
This will require some thought to design this right, so we should find out if OUIFR actually uses this.